src/WebJobs.Script/Models/ExtensionPackageReference.cs

```diff
@@ -1,16 +1,17 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
-
 namespace Microsoft.Azure.WebJobs.Script.Models
 {
     /// <summary>
     /// Represents a binding extension package reference.
     /// </summary>
-    public class ExtensionPackageReference
+    public class ExtensionPackageReference : ExtensionPackageRetrieve
+    {
+        public bool DeferAppActivationToServer { get; set; } = false;
```

Comment:
can we rename `DeferAppActivationToServer` to reflect setting online state. Mainly because the action is to bring host back online after install. `ActivationToServer` is a little confusing.

Reply:
Sorry, I think Delegate is the right word. I was thinking appActivation == bring app online (from the caller's perspective). DelegateAppActivationToServer? Open to suggestions.

---
In reply to: [299277937](https://github.com/Azure/azure-functions-host/pull/4617#discussion_r299277937) [](ancestors = 299277937)

Reply:
Modified this to use flags instead.